prow/plugins/project/project.go

```diff
@@ -113,23 +113,43 @@ func updateProjectNameToIDMap(projects []github.Project) {
 // processRegexMatches processes the user command regex matches and returns the proposed project name,
 // proposed column name, whether the command is to remove issue/PR from project,
 // and the error message
-func processRegexMatches(matches []string) (string, string, bool, string) {
-	var shouldClear = false
-	proposedProject := matches[1]
+func processRegexMatches(match string) (string, string, bool, string) {
```

Comment:
We can now rename this function to something like `processCommand(...)`

Reply:
done